src/Perlang.Tests/ConsoleApp/ProgramTest.cs

```diff
@@ -241,7 +241,7 @@ public void with_version_parameter_outputs_expected_value()
             public void with_eval_parameter_outputs_expected_value()
             {
                 // Arrange & Act
-                Program.MainWithCustomConsole(new[] { "-e", "print", "10" }, testConsole);
+                Program.MainWithCustomConsole(new[] { "-e", "print 10" }, testConsole);
```

Comment:
The way this test was formulated relied on some quite nasty semantics in `System.CommandLine`. While we are _still_ using a quite kludgy approach now, we should be able to get rid of it when incorporating the latest changes from upstream.

This does mean that you can no longer do cool stuff like `perlang -e 10 + 10`; you will _have_ to quote the expression as `perlang -e '10 + 10'`. However, the fact that we can use a more "standard" approach to arg parsing makes this a tradeoff I'm willing to do.